gno.land/pkg/sdk/vm/keeper_test.go

```diff
@@ -366,6 +366,16 @@ func main() {
 // Call Run with stdlibs.
 func TestVMKeeperRunImportStdlibs(t *testing.T) {
 	env := setupTestEnv()
+	testVMKeeperRunImportStdlibs(t, env)
+}
+
+// Call Run with stdlibs, "cold" loading the standard libraries
+func TestVMKeeperRunImportStdlibsColdStdlibLoad(t *testing.T) {
+	env := setupTestEnvCold()
+	testVMKeeperRunImportStdlibs(t, env)
```

Comment:
Should you run it twice?

Reply:
The intention is to have the same test, one time run with the "on-chain" stdlib loading flow and another time with the caching system (as most tests are). This is just to make sure that there is at least one test using the regular stdlib loading and attempting to import "std".

Reply:
Yep, I got this part, I was more asking if it would make sense to run several time the same helper. But maybe the helper itself is enough.